beetmoverscript/test/test_script.py

```diff
@@ -516,6 +526,7 @@ def test_get_destination_for_partner_repack_path(context, full_path,
     context.release_props = get_release_props(context)
     mapping_manifest = generate_beetmover_manifest(context)
 
+    # import pdb; pdb.set_trace()
```

Comment:
Let's remove this line. Even if its testing, debugging code should not make it in the final python package.